site/content/docs/5.2/components/navs-tabs.md

```diff
@@ -12,6 +12,13 @@ toc: true
 
 ## Base nav
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
```

Comment:
IMO we should place this callout after the full explanation. It's more understandable that this variant is not to be used because it's already explained in the info message.